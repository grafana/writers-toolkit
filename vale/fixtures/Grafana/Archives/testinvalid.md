Don't use:

- unarchive
- uncompress
- untar

- unzip

- ZIP file

- zip
- ZIP
