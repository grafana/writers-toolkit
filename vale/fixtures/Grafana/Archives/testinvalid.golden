testinvalid.md:3:3:Grafana.Archives:Use 'extract' instead of 'unarchive'.
testinvalid.md:4:3:Grafana.Archives:Use 'extract' instead of 'uncompress'.
testinvalid.md:5:3:Grafana.Archives:Use 'extract' instead of 'untar'.
testinvalid.md:7:3:Grafana.Archives:Use 'extract' instead of 'unzip'.
testinvalid.md:9:3:Grafana.Archives:Use 'archive' or 'compressed file' instead of 'ZIP file'.
testinvalid.md:11:3:Grafana.Archives:Use 'archive' or 'compressed file' instead of 'zip'.
testinvalid.md:12:3:Grafana.Archives:Use 'archive' or 'compressed file' instead of 'ZIP'.
